### PR TITLE
[NFC][ESIMD] Simplify simd_view template specialization for len=1

### PR DESCRIPTION
**Pull request by DenisBakhvalov**

### CODE REVIEW

**Comment by DenisBakhvalov** [comment]
@kychendev, please review.

**Review by kychendev — APPROVED** [review]

**Comment by DenisBakhvalov** [comment]
@kbobrovs is on vacation, but I hope this is a simple enough NFC patch and doesn't require his approval. 
@bader, please merge.

**Comment by bader** [comment]
@DenisBakhvalov, I suggest we add @kychendev to https://github.com/intel/llvm/blob/sycl/.github/CODEOWNERS#L101.

**Pull request merged**
